app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from __future__ import annotations
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_shortage = 0
+
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+
+        except VaccineError:
+            return "All friends should be vaccinated"
+
+        except NotWearingMaskError:
+            masks_shortage += 1
+
+    if masks_shortage != 0:
+        return f"Friends should buy {masks_shortage} masks"
+
+    else:
+        return f"Friends can go to {cafe.name}"
```

Comment:
no need to use else here

Reply:
@YuriiKindrat 
Yurii, good day!
Please, have a look at the updated solution "Solution updated" https://github.com/mate-academy/py-lockdown-cafe/pull/204/commits/4925df4e8086d3a7a09ac0e86c902c0ccd3b6549